.taskcluster.yml

```diff
@@ -56,6 +56,36 @@ tasks:
           owner: cdenizet@mozilla.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("build_documentation")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '2 hour'}
+        provisionerId: proj-relman
+        workerType: ci
+        payload:
+          maxRunTime: 3600
+          image: "rust:buster"
+          command:
+            - "/bin/bash"
+            - "-cx"
+            - "git clone --recursive --quiet ${repository} &&
```

Comment:
I wonder if we need a recursive clone for the docs

Reply:
Not for now, we can remove that option

Reply:
Sorry I didn't get your answer, can we or can we not remove the option?

Reply:
For now it's not necessary, but I guess that if we will add the `cargo publish` options it could be necessary for the documentation of the `tree-sitter` dependencies. Perhaps we could maintain it in order to be sure. Wdyt?

Reply:
OK, let's keep it, it doesn't hurt that much.